src/um/shobjidl_core.rs

```diff
@@ -95,3 +106,268 @@ interface ITaskbarList2(ITaskbarList2Vtbl): ITaskbarList(ITaskbarListVtbl) {
         fFullscreen: BOOL,
     ) -> HRESULT,    
 }}
+
+ENUM!{enum SLR_FLAGS {
+    SLR_NONE                        = 0,
```

Comment:
```suggestion
    SLR_NONE = 0,
```
Enums should not use aligned indentation.